doc/set-up-local-environment.md

```diff
@@ -0,0 +1,146 @@
+# Gridwich local development environment setup
+
+This article describes how to set up a local Gridwich development environment in either Visual Studio 2019 or above, or Visual Studio Code.
+
+## Prerequisites
+
+- Up-to-date [Visual Studio Code](https://code.visualstudio.com/) or [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/).
+- [Azure CLI](/cli/azure/install-azure-cli)
+- [.NET Core 3.1](https://dotnet.microsoft.com/download/dotnet-core/3.1)
```

Comment:
Just checking, is this still accurate?

<img width="1023" alt="image" src="https://github.com/mspnp/gridwich/assets/671075/330deec3-f715-44ed-acae-ca0018fc619d">


You can adjust this AFTER this merges, but want to make sure you're aware.

Reply:
Good catch, solution uses .NET 6